Modificaciones realizadas

libreec opened the pull request:
Un pr para todos los cambios realizados hasta ahora.

libreec committed:
Elimino el módulo l10n_ec pues el nombre se confunde con el módulo de odoo. En su lugar se creará l10n_ec_niifpymes.

libreec committed:
1.- Se eliminan varios archivos innecesarios para nuestra rama.
2.- Se modifican algunos archivos para nuestra rama que no deberán ser unidos a la rama oficial.
3.- Los archivos de l10n_ec_withdrawing/data serán reemplazados por otro módulo.

libreec committed:
Carga inicial del archivo de configuraciones.

libreec committed:
Carga inicial del módulo verificador que evita la duplicación de idenficación fiscal en los clientes o proveedores a menos que estén relacionadas como padre e hijo.

libreec committed:
Carga inicial del módulo para el igreso de los datos geopolíticos de Ecuador. Provincias, cantones y parroquias.

libreec committed:
Nuevas aclaratorias para el desarrollo.

Aclaratorias.

libreec committed:
Update README.md

libreec committed:
Corrección de errores en los nombres de los módulos.

libreec committed:
Merge branch '8.0' of https://github.com/libre-ec/odoo-ecuador into 8.0

libreec committed:
Se elimina el archivo ods de fuentes puesto que duplica la información y no es utilizado por el módulo.

libreec committed:
Carga inicial del módulo NIIF Pymes

libreec committed:
Cambios menores generales

libreec committed:
Borrado por error de datos en la carga inicial.

libreec committed:
Carga inicial de configuración de Niif Pymes

libreec committed:
Cambios generales.

libreec committed:
Cambios menores para facilitar la instalacion por modulo.

libreec committed:
Modificaciones al modulo para permitir su instalación en V8, no se ha verificado la funcionalidad.

libreec committed:
Migración a la nueva API, existe un error de instalación.

libreec committed:
Agregados nuevos módulos al instalador.

libreec committed:
Se cambia el campo ced_ruc por el campo vat para mantener compatibilidad con los demás módulos de Odoo.

libreec committed:
Cambios para reemplazar el campo ced_ruc con vat a fin de lograr compatibilidad con los demás módulos de odoo.

libreec committed:
Se comenta el código en el campo move_id que hacía que el campo se calcule solo. Existía el código pero no la función, por lo que se ha comentado. Verificar si es necesario para generar el código de reemplazo adecuado. Linea 135.

libreec committed:
Eliminado el código que no permitía la instalación del módulo.

libreec committed:
Cambios para instalar el módulo pos de Ecuador.

libreec committed:
Cambios para adaptar a la nueva api y correccion de errores. El módulo aún no se puede instalar.

ovnicraft commented on README.md:
```diff
@@ -1,36 +1,69 @@
-Localizacion de OpenERP para Ecuador
-====================================
+# Localizacion de OpenERP para Ecuador (Fábrica de Software Libre)
```
Es necesario el comentario entre () ?

libreec replied:
No, no es necesario, lo puse para evitar confusiones con la rama principal.

ovnicraft replied:
Los cambios aplicados van  la rama principal.

ovnicraft commented on README.md:
```diff
@@ -1,36 +1,69 @@
-requeridas para que OpenERP pueda implementarse en cualquier
-empresa ecuatoriana.
+Este proyecto es un derivado del proyecto odoo-ecuador, en el mismo se desarrollan y modifican los módulos con los cuales nuestra empresa aporta a la comunidad para que luego sean integrados a la rama oficial.
 
```
Creo que es un extra.

libreec replied:
El readme no está pensado para ser unido a la rama principal, todo ese documento está para diferenciar los dos desarrollos.

ovnicraft replied:
@libreec Si corremos el merge esto tambien se pondrá en la rama principal.

ovnicraft commented on README.md:
```diff
@@ -1,36 +1,69 @@
 
-Aclaratorio a la Licencia
--------------------------
+A diferencial de proyecto base, nosotros solo desarrollamos para la versión 8 y posteriores.
```
igual aca

ovnicraft commented on AUTHORS.txt:
```diff
@@ -1,2 +1,7 @@
 Milton Labanda <1000ton.lab@unl.edu.ec>
+
+# Fábrica de Software Libre
+
```
Desde la 3 esta por demas, mantengamos el formato regular.

ovnicraft commented on ec_id_unico/res_partner.py:
```diff
@@ -0,0 +1,40 @@
+                raise Warning(_(
+                    'Cada cliente o proveedor debe tener un número de identificación único a menos que estén relacionados. Los terceros que tienen el mismo número y no están relacionados son: %s!') % (', '.join(x.name for x in same_vat_partners)))
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Por qué esto no es parte de l10n_ec_partner ?

libreec replied:
Podríamos integrarlo, no lo puse directamente porque el módulo toma la base de uno de argentina y no quería mesclar el código, si deseas, lo no tengo problema en integrarlo.

ovnicraft replied:
@libreec si mucho mejor si se corrige

ovnicraft commented on l10n_ec_einvoice/einvoice.py:
```diff
@@ -34,22 +35,23 @@
+#    from suds.client import Client
+#    from suds.transport import TransportError
+#except ImportError:
+#    raise ImportError('Instalar Libreria suds')
```
esto es por  ? vi que tambien se elimino del archivo requirements.txt

libreec replied:
Me da un error al levantar el módulo por esta librería, no la tengo y no la encontré, por eso comenté el código. Ayudame para poder instalarla y evitar el error.

ovnicraft replied:
@libreec con pip install -r requirements.txt que esta en el repo se corrige.

ovnicraft commented on l10n_ec_einvoice/xades/sri.py:
```diff
@@ -29,14 +29,14 @@
+#    from suds.transport import TransportError
+#except ImportError:
+#    raise ImportError('Instalar Libreria suds')
+
```
Mismo problema con suds.

ovnicraft commented on l10n_ec_geopolitico/__openerp__.py:
```diff
@@ -0,0 +1,35 @@
+    'author': "Fábrica de Software Libre",
+    'website': "www.libre.ec",
+
+    'category': 'Contable',
```
Existe esta categoria ?

ovnicraft replied:
@alcidesrivera tenemos esto ya subido ?

alcidesrivera replied:
@ovnicraft @libreec si esta subido en l10n_ec_ote

ovnicraft commented on l10n_ec_geopolitico/__openerp__.py:
```diff
@@ -0,0 +1,35 @@
+	'views/res_partner_views.xml',
+	'data/res.country.state.csv',
+	'data/l10n_ec_geopolitico.canton.csv',
+#	'data/l10n_ec_geopolitico.parroquia.csv',
```
va a existir este archivo ?

ovnicraft commented on l10n_ec_geopolitico/__openerp__.py:
```diff
@@ -0,0 +1,35 @@
+#	'data/l10n_ec_geopolitico.parroquia.csv',
+
+    ],
+    # only loaded in demonstration mode
```
eliminarlo.

ovnicraft commented on l10n_ec_geopolitico/geopolitico.py:
```diff
@@ -0,0 +1,17 @@
+from openerp import models, fields
+
+class Canton(models.Model):
+    _name = 'l10n_ec_geopolitico.canton'
```
porque no utilizar el nombre de odoo para esto, por ejemplo `res.country.*`

ovnicraft commented on l10n_ec_geopolitico/geopolitico.py:
```diff
@@ -0,0 +1,17 @@
+    code = fields.Char(string="Código")
+
+class Parroquia(models.Model):
+    _name = 'l10n_ec_geopolitico.parroquia'
```
Misma observacion para el nombre del servicio.

ovnicraft commented on l10n_ec_invoice_sequence/__openerp__.py:
```diff
@@ -25,7 +25,8 @@
     'version' : '3',
-    'depends' : ['l10n_ec_withdrawing'],
+    'depends' : ['l10n_ec_withdrawing',
+                 'point_of_sale'],
```
no creo que sea necesario incluir a `point_of_sale` como requerimiento del modulo de secuencias.

ovnicraft commented on l10n_ec_withdrawing/__openerp__.py:
```diff
@@ -37,8 +37,8 @@
         'withdrawing_view.xml',
         'withdrawing_report.xml',
         'retention_wizard.xml',
-        'data/account.fiscal.position.csv'
```
por que eliminamos los archivos de datos ?

ovnicraft commented:
@libreec hay varias modificaciones que necesitan revisión

ovnicraft commented on l10n_ec_niif_pymes/__openerp__.py:
```diff
@@ -0,0 +1,32 @@
+    'installable': True,
+    'auto_install': False,
+    'application': False,
+}
```
no me queda claro que hace este modulo

ovnicraft commented:
@libreec el l10n_ec_ote se comento anteriormente y @alcidesrivera lo puso.
Puedes hacer rebase y actualizar tu repo.

ovnicraft commented on l10n_ec_withdrawing/withdrawing.py:
```diff
@@ -132,11 +132,14 @@ def _get_period(self, cr, uid, ids, fields, args, context):
+        '''
+        En la version original existe un compute a _get_move, pero no hay la funcion, por lo que se lo ha suprimido. Verificar si es necesario y reemplazar el codigo segun lo requerido.
+        #       compute='_get_move',
+        '''
```
es mejor poner el comentario en el commit

ovnicraft commented on l10n_ec_partner/partner.py:
```diff
@@ -68,20 +68,20 @@ def _check_ruc(self, partner):
 
     _columns = {
-        'ced_ruc': fields.char('Cedula/ RUC',
+        'vat': fields.char('Cedula/ RUC',
```
@alcidesrivera tu review al respecto

libreec committed:
Cambio del código a v8

libreec committed:
Se realizan cambios para que en lugar de elegir el tipo de persona se elija la posición fiscal y que, dentro de la posición fiscal, exista el campo tipo_persona. Así automatizamos el proceso de selección de tipo de persona al elegir la posición fiscal, misma que es importante tener siempre.

También se agrega un on_change y se cambia el tipo del campo en res_partner a fin de mantener compatibilidad.

ovnicraft commented:
Hasta un proximo update, lo cerramos
